public/blog/protocol-land-is-now-open-to-all.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Protocol.Land is now open to all'
+status: 'published'
+author:
+  name: 'Rohit Pathare'
+  avatar: 'https://avatars.githubusercontent.com/u/50047129?v=4'
+  twitter: 'https://twitter.com/ropats16'
+slug: 'protocol-land-is-now-open-to-all'
+category: 'ANNOUNCEMENT'
+description: "Protocol.Land announces its public launch, extending the benefits of decentralized code collaboration to everyone. Checkout what's in store and how to get started."
+coverImage: '/blog/images/protocol-land-is-open-to-all.png'
+transparentThumbnail: 'https://u2hyhgglc4egidq3zkthnzdruyatqwql5lk5foen3zv5zf5fo2wa.arweave.net/po-DmMsXCGQOG8qmduRxpgE4Wgvq1dK4jd5r3Jeldqw'
+themeColor: 'rgba(0, 0, 0, .15)'
+publishedAt: '2024-01-11'
+---
+
+## Protocol.Land: where decentralized protocols roam
+Last November, Protocol.Land launched in beta as a decentralized code collaboration platform. This launch marked a shift towards a future where transparency and permanence are embedded in every line of code. Since then, we’ve added features like encrypted private repositories, static page deployments, and aliases for these deployments. We've also introduced built-in bounties for code contributions, further incentivizing collaborative development.
+
+At Protocol.Land, we are on a mission to empower developers by ensuring the preservation and evolution of their work. This ensures  future builders and even future generations can build upon a legacy of shared knowledge.
+
+Today, we're excited to announce the public launch of Protocol.Land, extending the benefits of decentralized code collaboration to everyone!
+
+Here’s what Protocol.Land has in store:
+
+### In sync with Git
+Protocol.Land integrates seamlessly with Git, keeping your typical workflow from repository creation to pushing changes remains intact, blending the new with the familiar in a streamlined experience.
+
+### Rewarding contributions
+Protocol.Land introduces an incentivized programming environment. You can assign bounties to issues, rewarding contributors for their efforts and acknowledging the hard work poured into every line of open source code.
+
+### Your code on ice
+Your code is preserved indefinitely, stored in over 1000 locations worldwide on the Arweave network, instantly. Protocol Land is your personal arctic code vault, ensuring eternal preservation.
+
+### A snapshot with every commit
+Your code isn't just archived; it's a living repository on Protocol.Land. Every commit is a permanent, on-chain snapshot. This approach provides a complete, verifiable history of your project, essential for reliable progress tracking and validating its integrity.
+
+### Timeless frontends
+Deploy and perpetually host your frontends on the most distributed and decentralized network, all at zero upkeep cost. Protocol.Land also enables user-friendly aliases for these deployments using the Arweave Name System (ArNS).
```

Comment:
Link to ArNS ?

Reply:
Did not want to direct our audience to external protocols for the announcement